core/src/main/clojure/xtdb/sql/plan.clj

```diff
@@ -1629,11 +1629,27 @@
           data-type (-> (.dataType ctx) (.accept (->CastArgsVisitor env)))]
       (handle-cast-expr ve data-type)))
 
+  (visitCollateExpr [_ ctx]
+    ;; TODO - apply collation
+    (log/trace "visitCollateExpr" (.getText (.collation ctx)))
+    (.getText (.collation ctx)))
```

Comment:
I doubt the right thing to do is return the collation part here, even if we're not planning to fully implement it. Once implemented, won't this expression return the `expr` string rather than the name of the collation?

tbh I'm surprised this doesn't break something else?